client/jetpack-connect/authorize.js

```diff
@@ -704,15 +741,25 @@ export class JetpackAuthorize extends Component {
 	}
 
 	render() {
+		const { translate } = this.props;
+		const wooDna = this.getWooDnaConfig();
 		return (
-			<MainWrapper isWoo={ this.isWooOnboarding() }>
+			<MainWrapper
+				isWoo={ this.isWooOnboarding() }
+				wooDna={ wooDna }
+				pageTitle={ wooDna && wooDna.name( translate ) + ' — ' + translate( 'Connect' ) }
+			>
 				<div className="jetpack-connect__authorize-form">
 					<div className="jetpack-connect__logged-in-form">
 						<QueryUserConnection
 							siteId={ this.props.authQuery.clientId }
 							siteIsOnSitesList={ this.props.isAlreadyOnSitesList }
 						/>
-						<AuthFormHeader authQuery={ this.props.authQuery } isWoo={ this.isWooOnboarding() } />
+						<AuthFormHeader
+							authQuery={ this.props.authQuery }
+							isWoo={ this.isWooOnboarding() }
+							wooDna={ this.getWooDnaConfig() }
```

Comment:
Nitpick: could this be `wooDna={ wooDna }`?

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/41798/commits/e75fab9377165f088ad48456aa7828182f562366